README.md

```diff
@@ -273,15 +273,18 @@ After executing `make` you will have sample applications in your `build/samples`
 #### Sample: kvsWebrtcClientMaster
 This application sends sample H264/Opus frames (path: `/samples/h264SampleFrames` and `/samples/opusSampleFrames`) via WebRTC. It also accepts incoming audio, if enabled in the browser. When checked in the browser, it prints the metadata of the received audio packets in your terminal. To run:
 ```shell
-./samples/kvsWebrtcClientMaster <channelName>
+./samples/kvsWebrtcClientMaster <channelName> <storage-option> <audio-codec> <video-codec>
 ```
 
 To use the **Storage for WebRTC** feature, run the same command as above but with an additional command line arg to enable the feature.  
 
 ```shell
-./samples/kvsWebrtcClientMaster <channelName> 1
+./samples/kvsWebrtcClientMaster <channelName> 1 <audio-codec> <video-codec>
```

Comment:
When using Safari 17.4.1 with h265 enabled as JS viewer and C SDK as master with opus/h265 the video playback on JS side is constantly jumping back.

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/assets/7199741/55ff3d31-38d0-4fee-aa43-2860ce9180da

Reply:
There was an issue with the source frames. Refer https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/1978